README.md

```diff
@@ -1,24 +1,74 @@
-# Animated-name
+<h1 align="center"> ANIMATED NAME | HACKTOBERFEST </h1>
 
-## An Animated name with html and css
-An animated names with html and css with different style and color and animation 
+**A repo showing animated name using HTML and css**
 
-##  Hacktoberfest
-###  Add your own name with your name(contributor name )
-Like:
+###  Add your details to the [names.js](/names.js) file.
+The details include:
+```
+1. Your name
+2. Your Full Name
+3. Your GitHub url
+```
 
-i am yourname 
 
-made by yourname with link to your github link in that part
+## 🛠️ Installation Steps
 
-Different section with Different contributor
+**1**. [Fork](https://github.com/Rajkumar-justcoder/Animated-name/fork) the project. Click on the <a href="https://github.com/Rajkumar-justcoder/Animated-name/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="15" width="15"></a> icon in the top right to get started
 
 
-## How to contribute 
-step 1: Create issue
+**2**. Clone the project, you can use the following command:
 
-step 2: Fork repo. make changes. make new branch,push code 
+```bash
+Using HTTPS
+git clone https://github.com/<your-github-username>/Animated-name
+or using SSH
+git clone git@github.com:<your-github-username>/Animated-name.git
+```
+**3**. Navigate to the project directory
 
-step 3: Make pr!!
+```bash
+cd Animated-name
+```
+## 🥂 After making a change
 
-Note: all Hacktoberfest rules should be followed 
+**1**. Create a new branch
+
+```bash
+git checkout -b YourBranchName
+```
+
+**2**. Add it to staging area
+
+
+```bash
+git add <path to the file you worked on>
+```
+
+**3**. Commit your changes with
+
+```bash
+git cz
```

Comment:
whats this git cz ?

Reply:
ig its for commit msg then used do  preffered way

Reply:
Done :heavy_check_mark:  @Rajkumar-justcoder